strongbox-data-service/src/main/resources/etc/conf/db_EMBEDDED.yaml

```diff
@@ -0,0 +1,7 @@
+strongbox:
```

Comment:
Do we have documentation for these configuration files in our wiki? :)

Reply:
most likely we don't have it.

Reply:
If we don't, and, if we decide to still proceed with Janusgraph, we'll need to have follow-up tasks for documenting these as well.

Reply:
Do we have the follow-up tasks for documenting this?